### ISSUE TITLE
Bugs in cipher message test cases 

### ISSUE THREAD

**Issue by jphartmann** [rbowler/spinhawk#54]
Taking the km* test cases to real iron (z12) shows:

KM fc52 shows an incorrect expected result.  Hardware gives 
```
r 600.10
00000600 52F67E0A 657CDCE5 A26ADC57 422AC064
```
kmo fc 3 similar:
```
r 600.8
00000600 C8A06C9E FF17D58A
```
Several fc 3 tests show fc2 in their initial comment.

**Comment by jphartmann** [comment]
script ../spinhawk/tests/kmctr28.txt*** stack smashing detected ***: /usr/data/src/hercules/shdbg/.libs/lt-hercules terminated

**Comment by jphartmann** [comment]
While Hyperion crypto has diverged from Spinhawk, the bug is there too.  The parameter block should be 64 bytes which is the cause of the storage overlay.  You will likely find that the result is incorrect.  I replaced his hokey calculations with tables to get the same results as a z12.

**Comment by jphartmann** [comment]
None of the KMCTR test cases try to decrypt.

**Comment by rbowler** [comment]
Storage overlay fixed by Jürgen Winkelmann in commit aaff27ddddbcae1d406d0ce1d86ca009581c8f7c
